src/main/java/subway/ui/LineController.java

```diff
@@ -17,21 +18,16 @@
 @RequestMapping("/lines")
 public class LineController {
     private final LineService lineService;
-    private final StationService stationService;
 
-    public LineController(LineService lineService, StationService stationService) {
+    public LineController(LineService lineService) {
         this.lineService = lineService;
-        this.stationService = stationService;
     }
 
     @PostMapping
     public ResponseEntity<LineResponse> createLine(@RequestBody LineRequest lineRequest) {
-        Station upStation = stationService.findStation(lineRequest.getUpStationId());
-        Station downStation = stationService.findStation(lineRequest.getDownStationId());
+        Line newLine = new Line(lineRequest.getName(), lineRequest.getColor(), lineRequest.getDistance());
+        Line savedLine = lineService.saveLine(newLine, lineRequest.getUpStationId(), lineRequest.getDownStationId());
```

Comment:
```suggestion
        Line savedLine = lineService.saveLine(lineRequest);
```

이렇게 처리해도 되지 않을까요?
컨트롤러에게 도메인 객체를 생성하는 일을 맡는게 적절해보이진 않네요.